packages/hydrooj/src/handler/problem.ts

```
@@ -646,28 +646,15 @@ export class ProblemFilesHandler extends ProblemDetailHandler {
     notUsage = true;
 
     @param('d', Types.CommaSeperatedArray, true)
-    @param('pjax', Types.Boolean)
     @param('sidebar', Types.Boolean)
-    async get(domainId: string, d = ['testdata', 'additional_file'], pjax = false, sidebar = false) {
+    async get({ }, d = ['testdata', 'additional_file'], sidebar = false) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)